feat(draft3): add support for draft3 (#270)
sourcemeta/learnjsonschema.com: content/draft3/core/extends.markdown

---
keyword: "extends"
signature: "URI Reference | Schema | Array<Schema>"

> value: This keyword must be set to a valid JSON Schema which provides a base schema which the current schema will inherit from. The inheritance rules are such that any instance that is valid according to the current schema MUST be valid according to the referenced schema. This MAY also be an array, in which case, the instance MUST be valid for all the schemas in the array. A schema that  extends another schema MAY define additional attributes, constrain existing attributes, or add other constraints. 

Reviewer: I think we can simplify this. What about:

```suggestion
value: This keyword must be set to either a schema, a list of schemas, or a URI that points to a schema
```

> summary: "An instance validates successfully against this keyword if it validates successfully against all schemas defined by this keyword's value. If set to a URI Reference, it must be set to an absolute URI or a relative reference as defined by [RFC 3986](https://www.rfc-editor.org/info/rfc3986)"

Reviewer: What about this:

```suggestion
summary: "Validation succeeds if the instance is valid against the schema or schemas defined by this keyword"
```

kind: [ "applicator" ]
instance: [ "any" ]

> specification: "https://json-schema.org/draft-04/draft-fge-json-schema-validation-00#rfc.section.5.5.3"

Reviewer: Old draft links?
